service/security/shiro/src/test/resources/features/AccessInfoService.feature

```diff
@@ -328,14 +359,33 @@ Scenario: Count access role entities by scope
     result list must be returned. No exception must be thrown.
 
     Given A scope with ID 10
+    And A user such as
+        | name     | displayName     | email              | phoneNumber     |
+        | kapua-u1 | Kapua User 1    | kapua_u1@kapua.com | +386 31 323 555 |
+    When I create the access info entity
+    And The permissions "read, write, execute"
+    And The role "test_role_1"
     And I create the access role
+    And The role "test_role_2"
     And I create the access role
+    And The role "test_role_3"
     And I create the access role
+    
     Given A scope with ID 20
+    And A user such as
+        | name     | displayName     | email              | phoneNumber     |
+        | kapua-u2 | Kapua User 2    | kapua_u2@kapua.com | +386 31 323 555 |
+    When I create the access info entity
```

Comment:
I've just changed scenarios that was failing because of new integrity checks (i.e. check that the `roleId` and `accessInfoId` exists when creating an `accessRole`. 
So the scenarios are the same as before as (I think) I changed only preconditions of scenarios, not the final checks. 
Are you suggesting to add also negative test scenarios?